src/pepr/operator/controllers/istio/injection.ts

```diff
@@ -20,9 +20,9 @@ export async function enableInjection(pkg: UDSPackage) {
   const annotations = sourceNS.metadata?.annotations || {};
   const pkgKey = `uds.dev/pkg-${pkg.metadata.name}`;
 
-  // Save the original value of the istio-injection label only if it's not already set
-  if (!annotations[injectionLabel]) {
-    annotations[injectionAnnotation] = labels[injectionLabel] || "non-existent";
+  // If the original namespace injectionLabel did not exist, mark it as non-existent in the annotations
+  if (!labels[injectionLabel]) {
```

Comment:
changed this because it wasn't connected to anything on the other end - can change it back if that was intended

Reply:
I think the logic is weird but seems right? Intent was to be able to revert the label back to its original value [here](https://github.com/defenseunicorns/uds-core/blob/9518634b24f2d5c285e598f8620849bbc6288ba4/src/pepr/operator/controllers/istio/injection.ts#L69-L77).

What we want is to revert the label to its original value if set OR delete it if it was non-existent. So I think we still need to set the annotation to `labels[injectionLabel]` if that has a value, or `non-existent` if it doesn't. TLDR think we can revert this change and it should work as intended?